rake_build/turn_csv_to_popolo.rb

```diff
@@ -13,8 +13,7 @@
 
   task :meta_info => :load do
     @json[:meta] ||= {}
-    # TODO: allow for more than one source
-    @json[:meta][:sources] = instructions(:sources).map { |s| s[:source] }.compact.uniq
+    @json[:meta][:sources] = instructions(:sources).map { |s| s[:source] }.compact.uniq.flatten
```

Comment:
:+1:

Though to be slightly neater you could do this with `instructions(:sources).flat_map { |s| s[:source }…` rather than a separate `flatten` step.

NB: You should also [close the issue from the commit](https://help.github.com/articles/closing-issues-via-commit-messages/).